src/modules/uiKit/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  Pagination as MUIPagination,
+  PaginationItem,
+  PaginationItemProps,
+  PaginationProps,
+} from '@material-ui/lab';
+import { usePaginationStyles } from './usePaginationStyles';
+
+export const Pagination = (props: PaginationProps) => {
+  const classes = usePaginationStyles();
+
+  const itemCreator = (itemProps: PaginationItemProps) => (
```

Comment:
```suggestion
  const renderItem = (itemProps: PaginationItemProps) => (
```

Let's keep it `renderItem` to not add variet of namings